sqlglot/dialects/snowflake.py

```diff
@@ -143,6 +143,14 @@ def _datatype_sql(self: generator.Generator, expression: exp.DataType) -> str:
     return self.datatype_sql(expression)
 
 
+def _parse_convert_timezone(args: t.Sequence) -> exp.Expression:
+    if len(args) == 3:
+        converted: t.List[exp.Expression] = [maybe_parse(arg, dialect=Snowflake) for arg in args]
+        return exp.Anonymous(this="CONVERT_TIMEZONE", expressions=converted)
```

Comment:
exp.func("CONVERT_TIMEZONE", expression=...)

Reply:
I initially tried `exp.func`, but since `CONVERT_TIMEZONE` is in the `parser.FUNCTIONS` dict, it ends up calling this function again recursively on line 5414 of expressions.py and causing a recursion loop. (that's also why I was calling maybe_parse - I was looking at the body of `exp.func`, but it makes sense that it's not necessary here now)